tests/testmakespreadsheet.py

```diff
@@ -9,7 +9,7 @@
 ## Define tests to run here!!!
 tests = [
 'makespreadsheet',
-#'unittests',
+'unittests',
```

Comment:
the problem with the unittests is that they kill the console (at least for me...?)